src/index.js

```diff
@@ -4,58 +4,51 @@ import './index.css';
 
 function Square(props) {
 
-  if (props.value === 'X') {
     return (
       <button 
-        className={"square X " + (props.Winning ? "victory-square" : null)} 
+        className={"square " + (props.Winning ? "victory-square " : null) + (props.value === 'X' ? " X" : " O")} 
         onClick={props.onClick}
       >
         {props.value}
       </button>
     );
-  } else {
-    return (
-      <button 
-        className={"square O " + (props.Winning ? "victory-square" : null)} 
-        onClick={props.onClick}
-      >
-        {props.value}
-      </button>
-    );
-  }
 }
 
 //Board
 
 class Board extends React.Component {
   
-  renderSquare(i) {
-    return (<Square 
-      Winning = {this.props.winningSquares.includes(i)}
-	    value={this.props.squares[i]} 
-	    onClick={() => this.props.onClick(i)} 
-	  />
-    );
+  makeRow(x) {
+    return <div className="board-row">{x}</div>;
   }
 
+  makeBoard(n) {
```

Comment:
It doesn't depend on the context, so it can be moved to a function.

`function generateBoardMatrix(size)`

Reply:
Fixed, thanks